### PR TITLE
Set overflow property on document to avoid scrolling

### PR DESCRIPTION
**Pull request by nickoola**

### CODE REVIEW

**Comment by nickoola** [comment]
Tested on Kairos !

File: src/modal/index.tsx [hunk 1]
```diff
@@ -53,23 +53,33 @@ class Modal extends Component<ModalProps> {
...
 
   componentDidUpdate(prevProps: ModalProps) {
     if (!prevProps.isOpen && this.props.isOpen) {
       this.addListeners()
+      this.setDocumentScroll('hidden')
     }
 
     if (!this.props.isOpen && prevProps.isOpen) {
       this.removeListeners()
+      this.setDocumentScroll('')
```
Comment (Tuxosaurus) [inline src/modal/index.tsx]: Shouldn't you set the overflow to "visible"?

Comment (nickoola) [reply src/modal/index.tsx]: This way it remove the property

**Comment by nickoola** [comment]
Tested on Safari and it works !

**Pull request merged**